corehq/apps/userreports/tests/test_expressions.py

```diff
@@ -911,6 +911,7 @@ def test_invalid_eval_expression(self, source_doc, statement, context):
     ("range(1, a)", {"a": 5}, [1, 2, 3, 4]),
     ("a or b", {"a": 0, "b": 1}, True),
     ("a and b", {"a": 0, "b": 1}, False),
+    ("int(10 in range(1,20))", {"a": 2}, 1),
```

Comment:
Guessing htis isn't supported because range returns an iterator in py3? I'm hesitant to remove functionality since we don't have a great way to see if people are using this